src/edge/importer.py

```diff
@@ -107,9 +107,28 @@ def build_fragment(self):
         prev = None
         flen = 0
         seqlen = len(self.__sequence)
+
+        cs_limit = 1000000
```

Comment:
adding a comment here to briefly describe the rationale of breaking up the big chunk sizes into smaller sizes?

Reply:
Thanks! This is super clear now!